### ISSUE TITLE
T2 support for Thrustmaster T-Flight Hotas One

### ISSUE THREAD

**Issue by STE6256** [J2Kbr/GtunerIV#313]
Currently T2 sees Thrustmaster T-Flight Hotas One as a regular XBOX ONE controller and not a HOTAS.
In order for HOTAS support on XBOX to work correctly need Thrustmaster T-Flight Hotas One in T2 so we can remap keys with other PC HOTAS

**Comment by swoodbury1963** [comment]
Same for T-Flight HOTAS for PS4. I'm want to configure a Thrustmaster T.1600M FCS for PS4 with Elite Dangerous.

**Comment by J2Kbr** [comment]
HID configuration files for:

Thustmaster T.Flight Hotas
and
Thustmaster T.Flight Hotas One

were added to Gtuner IV version 1.02 (to be release shortly).

**Comment by CaptMoore** [comment]
The update did not appear to work for me, as far as I can tell nothing changed when using the new firmware despite adding HID config files.

**Comment by swoodbury1963** [comment]
I've been trying on PS4 myself. I've tried mapping the throttle, paddles to different axis as well as the Joystick twist. I've tried mapping them to "left stick axis" as well as the Accel Axis, none of them register when trying to assign them in Elite Dangerous controller options. Buttons work, the only axis that works is the default Joystick X/Y axis. I've also tried the Thrustmaster Duo git files without success.
On the T2 I've tried Joystick in A, Throttle in Prog port, PS4 controller wired in B and Visa Versa.
I've tried wireless BT with green lights all around. Throttle in B and swapping Joystick to the Prog port via OTG cable.
